pkg/components/components.go

```diff
@@ -785,7 +785,8 @@ func GetCSVBase(name, namespace, displayName, description, image, replaces strin
 				"description":    description,
 				"repository":     "https://github.com/kubevirt/hyperconverged-cluster-operator",
 				"support":        "false",
-				"operatorframework.io/suggested-namespace": namespace,
+				"operatorframework.io/suggested-namespace":       namespace,
+				"operators.openshift.io/infrastructure-features": "[\"Disconnected\"]",
```

Comment:
I would get rid of the confusing `\"` by wrapping it with back quotes; something like:

```golang
"operators.openshift.io/infrastructure-features": `["Disconnected"]`,
```

Reply:
Thanks, done.